CRTX-243111 | Mapping for Menlo Security (#44897)
demisto/content: Packs/MenloSecurity/ReleaseNotes/1_0_2.md

#### Modeling Rules

##### New: Menlo Security Modeling Rule

> - New: Added Menlo Security Modeling Rule to enable normalization of Menlo Security data ingested via API into Cortex XSIAM for investigation and analysis.

Reviewer: ```suggestion
- Added a new Menlo Security Modeling Rule to enable normalization of Menlo Security data ingested into Cortex XSIAM for investigation and analysis.
```
